test/bundles/validNodeBundleWithModulesAsArray.js

```diff
@@ -0,0 +1,6 @@
+exports.ids = ["common"];
+exports.modules = {
```

Comment:
Filename is "validNodeBundle**WithModulesAsArray**.js" but modules are in the object. Shall we remove `WithModulesAsArray` part?